fontbe/src/features/marks.rs

```diff
@@ -622,8 +624,8 @@ impl<'a> MarkLookupBuilder<'a> {
             gsub.as_ref(),
         )?;
         // https://github.com/googlefonts/ufo2ft/blob/5a606b7884bb6da/Lib/ufo2ft/featureWriters/markFeatureWriter.py#L1156
-        // I do not understand why we have all this fancy logic when it feels like
-        // 'not abvm' is implicit?
+        // TK: there's another bug here I think!? we can't trust char map, need
```

Comment:
In the course of writing this PR I noticed another thing that looks like a bug, will have a patch for this as a followup.